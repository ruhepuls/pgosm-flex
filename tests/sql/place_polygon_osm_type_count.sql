SELECT osm_type COLLATE "C", COUNT(*)
    FROM osm.place_polygon 
    GROUP BY osm_type COLLATE "C"
    ORDER BY osm_type COLLATE "C"
;