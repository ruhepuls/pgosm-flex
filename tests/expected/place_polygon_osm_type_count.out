admin_level|1
archipelago|1
boundary|12
island|6
islet|1
neighborhood|1
neighbourhood|110
plot|3
square|4
suburb|9
town|1
